wirecard/oxid-ee
# WDPD-216 Improve payment method title translation

### This PR

* Adds initial payment method title in metadata database table
* Merchant can change payment method title which will be reflected in payment methods list in admin, and on frontend
* Initial payment method title always remains unchanged and it is shown in payment method configuration page in admin
* Adds three new translation keys for payment methods which include "guaranteed" in their titles. New keys have sufix "_custom" and have the same translation, but just without wording "guaranteed/garantiert"

### How to Test

* Activate the module
* Go to Shop Settings -> Payment methods
* Change payment method title
* New payment method title can be visible in payment method list, and initial payment method title (which is shown in payment method configuration section) remains unchanged

### Jira Ticket
https://jira.parkside.at/browse/WDPD-216
